app/backend/approaches/approach.py

```diff
@@ -205,6 +207,10 @@ async def search(
     def get_sources_content(
         self, results: List[Document], use_semantic_captions: bool, use_image_citation: bool
     ) -> list[str]:
+
```

Comment:
I moved this function out of text.py since it was a 2-line file, and the function is only used in this one method.